src/wordcloud2.js

```diff
@@ -836,6 +863,82 @@ if (!window.clearImmediate) {
       })
     }
 
+    var getHtmlTextInfo = function getTextHeight(font, size, text, info) {
+      info = info || {};
+
+      var container = document.createElement('div')
+      var inline = document.createElement('span')
+      var block = document.createElement('div')
+      var ascender, descender
+
+      container.style.position = 'relative'
+      // make it look a little prettier if debugging...
+      container.style.marginTop = '0.8rem'
+      container.style.marginBottom = '1rem'
+      // necessary when size is smaller than default/computed size...
+      container.style.lineHeight = '0'
+      container.style.fontSize = '0px'
+      // ensure longer entries don't wrap...
+      container.style.whiteSpace = 'nowrap'
+
+      inline.style.position = 'relative'
+      inline.style.zIndex = '100'
+      inline.textContent = text
+      inline.style.fontFamily = font
+      inline.style.fontSize = size + 'px'
+      inline.style.lineHeight = 'normal' // equivlent to wordcloud
+
+      block.style.display = 'inline-block'
+      block.style.width = '1px'
+      block.style.height = '0px'
+      block.style.verticalAlign = 'baseline'
+
+      container.appendChild(inline)
+      container.appendChild(block)
+ 
+      document.body.appendChild(container)
```

Comment:
Shouldn't the `container` be the child of the "canvas" `<div>` so that it would inherit the style?